test/json_templates/bucket.json

```diff
@@ -1,8 +1,13 @@
 {
+{{if eq .TFVersion "0.12"}}
```

Comment:
What about future 0.13+ versions?

Reply:
if TF13+ will require it's own changes, probably we'll have to add another value for variable, currently we support 11 and 12